app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -290,19 +294,22 @@ class UserPreference : AppCompatActivity() {
                     summary = getString(R.string.biometric_auth_summary_error)
                 } else {
                     setOnPreferenceClickListener {
+                        isEnabled = false
```

Comment:
This disable/enable dance is to prevent against people spamming the toggle and leaving the Authenticator in a state disjoint from the toggle which can create a bypass for biometric authentication.